### PR TITLE
[Dont Merge] ServiceFabric_StartupServicesFile_Support

### PR DESCRIPTION
**Pull request by sunil-indoria**

### CODE REVIEW

File: SkipStrongName.ps1 [hunk 1]
```diff
@@ -10,7 +10,8 @@ $assemblies = "FabActUtil",
               "Microsoft.ServiceFabric.Services.Wcf",
               "Microsoft.ServiceFabric.Actors.Tests",
               "Microsoft.ServiceFabric.Services.Tests",
-              "Microsoft.ServiceFabric.Services.Remoting.Tests"
+              "Microsoft.ServiceFabric.Services.Remoting.Tests",
+			  "StartupServicesUtility"
```
Comment (anantshankar17) [inline SkipStrongName.ps1:14]: Nit: Fix indentation.

File: properties/service_fabric_managed_netframework.targets [hunk 2]
```diff
@@ -2,7 +2,8 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <!-- List of .csproj to build -->
-    <Projects_Prod_netframework Include="$(RepoRoot)src\FabActUtil\FabActUtil.csproj;
+    <Projects_Prod_netframework Include="$(RepoRoot)src\StartupServicesUtility\StartupServicesUtility.csproj;
+								   $(RepoRoot)src\FabActUtil\FabActUtil.csproj;
```
Comment (anantshankar17) [inline properties/service_fabric_managed_netframework.targets:6]: Nit: Fix indentation everywhere, in all files.

File: properties/service_fabric_managed_netstandard.targets [hunk 3]
```diff
@@ -3,7 +3,8 @@
   
   <!-- Target to build net standard.-->
```
Comment (anantshankar17) [inline properties/service_fabric_managed_netstandard.targets:4]: You have not packaged the new DLL in the nuget package. You need to update the nuproj files.

Comment (sunil-indoria) [reply properties/service_fabric_managed_netstandard.targets:4]: Fixed intendation and nuget addition. Not sure if I need to keep all the target paths in StartupServicesUtility.nuget or keeping netstandard2.0 and net45 should be enough.

**Comment by sunil-indoria** [comment]
These changes have been covered and merged as part of another PR.

**Pull request closed without merging**